migration-engine/migration-engine-tests/tests/migrations/indexes.rs

```diff
@@ -272,19 +271,8 @@ async fn index_renaming_must_work(api: &TestApi) -> TestResult {
 
     // Test that we are not dropping and recreating the index. Except in SQLite, because there we are.
     if !api.is_sqlite() {
-        let expected_steps = &[
-            SqlMigrationStep::AlterIndex(AlterIndex {
-                table: "A".into(),
-                index_new_name: "customNameA".into(),
-                index_name: "customName".into(),
-            }),
-            SqlMigrationStep::AlterIndex(AlterIndex {
-                table: "A".into(),
-                index_new_name: "customNameNonUniqueA".into(),
-                index_name: "customNameNonUnique".into(),
-            }),
-        ];
-        let actual_steps = result.sql_migration();
+        let expected_steps = &["AlterIndex", "AlterIndex"];
```

Comment:
Isn't this testing a bit less than before now?

Reply:
Same with the other changes down below too...

Reply:
Yep, but that's ok because we'll test the exact process in create_migration tests. Users will also catch regressions here since they see the migrations. It's a necessary evil but I really really want SqlMigration to be private.